pkg/trace/info/stats.go

```diff
@@ -288,6 +284,51 @@ func (s *SpansMalformed) String() string {
 	return mapToString(s.tagValues())
 }
 
+// maxAbsPriority specifies the absolute maximum priority for stats purposes. For example, with a value
+// of 10, the range of priorities reported will be [-10, 10].
+const maxAbsPriority = 10
+
+// samplingPriorityStats holds the sampling priority metrics that will be reported every 10s by the agent.
+type samplingPriorityStats struct {
+	// counts holds counters for each priority in position maxAbsPriorityValue + priority.
+	// Priority values are expected to be in the range [-10, 10].
+	counts [maxAbsPriority*2 + 1]int64
+}
+
+// CountSamplingPriority increments the counter of observed traces with the given sampling priority by 1
+func (s *samplingPriorityStats) CountSamplingPriority(p sampler.SamplingPriority) {
+	if p > (-1*maxAbsPriority) && p < maxAbsPriority {
```

Comment:
```suggestion
	if p >= (-1*maxAbsPriority) && p <= maxAbsPriority {
```
Shouldn't it be like this for `[-10, 10]` ?

Reply:
Yes, thanks a million for catching this. I have fixed it and improved the unit test to cover all possible values.